massa-pos-exports/src/pos_final_state.rs

```diff
@@ -1734,37 +1811,5 @@ mod tests {
         assert_eq!(cycles.len(), 1, "wrong number of cycles");
         assert_eq!(cycles[0].0, 0, "cycle should be the 1st one");
         assert_eq!(cycles[0].1, false, "cycle should not be complete yet");
-
-        let cycle_info_a = pos_state.get_cycle_info(0);
-
-        let mut prod_stats = HashMap::default();
-        prod_stats.insert(
-            addr,
-            ProductionStats {
-                block_success_count: 12,
-                block_failure_count: 18,
-            },
-        );
-
-        let cycle_info_b = CycleInfo::new_with_hash(
-            0,
-            false,
-            BTreeMap::default(),
-            bitvec![u8, Lsb0; 0, 0, 0, 1, 1, 0, 0, 1],
-            prod_stats,
-        );
-
-        assert_eq!(
-            cycle_info_a.roll_counts_hash, cycle_info_b.roll_counts_hash,
-            "roll_counts_hash mismatch"
-        );
-        assert_eq!(
-            cycle_info_a.production_stats_hash, cycle_info_b.production_stats_hash,
-            "production_stats_hash mismatch"
-        );
-        assert_eq!(
-            cycle_info_a.cycle_global_hash, cycle_info_b.cycle_global_hash,
-            "global_hash mismatch"
-        );
```

Comment:
I think we still want to test that the values (e.g. production stats) are computed correctly. Why did the test fail here?

Reply:
for some reason, "cycle_global_hash", "cycle_global_hash" etc... did not exist anymore

Reply:
That's weird because the current testnet24 branch seems to have them and the CI passes: https://github.com/massalabs/massa/pull/4001/files#diff-999f2be25e86f9ed087021cc5ae7d3e84af75602f84bfa60f6284590c2283958

This issue will remove them: https://github.com/massalabs/massa/issues/4153, but it's not merged yet

Reply:
Maybe it's because of conflicts with this one: https://github.com/massalabs/massa/pull/4104, can you check again please? :)

Reply:
Any idea on what is happening ?

Reply:
I reinstated the removed stuff, re-merged testnet24 in this branch. Does not compile

Reply:
My bad I had screwed up. Corrected :) thanks for spotting it